docs/yaml-classes-and-relationships.rst

```diff
@@ -569,7 +590,7 @@ dynamicview_weight
   :Description: Dynamic View weight for objects of this class. Higher numbers are further to the right. Can be overridden by implementing getDynamicViewGroup() method on class.
   :Required: No
   :Type: float or int
-  :Default: 1000 + (order * 100)
+  :Default: 1000 * (1 + order/100)
```

Comment:
This formula is confusing to me. Though it looks to match what the code does. Perhaps it would be a bit easier to understand if stated as `1000 + (order * 10)`.